packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1595,6 +1595,9 @@ export default class TradeStore extends BaseStore {
             });
         }
         if ('active_symbols' in req) {
+            if (this.root_store.client.loginid && Object.keys(this.root_store.client.accounts || {}).length) {
```

Comment:
```suggestion
            if (this.root_store.client.is_logged_in) {
```
@henry-deriv please try this🙏🏻

Reply:
works!